tests/platform_tests/mellanox/check_sysfs.py

```diff
@@ -163,6 +163,10 @@ def check_sysfs(dut):
 
     logging.info("Check SFP related sysfs")
     for sfp_id, sfp_info in list(sysfs_facts['sfp_info'].items()):
+        # Skip the check for empty values
+        if not sfp_info["temp_fault"]:
+            continue
+
```

Comment:
Is `sfp_info["temp_fault"] == None` when the sfp is missing?

Reply:
It's an empty string.